### PR TITLE
Fix unused variable found

### PR DESCRIPTION
**Pull request by edgarriba**
### Description

Fixes https://deepsource.io/gh/kornia/kornia/issue/PYL-W0612/occurrences?page=1

### Status
**Ready/Work in progress/Hold**

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or new feature that would cause existing functionality to change)
- [ ] New tests added to cover the changes
- [ ] Docstrings/Documentation updated

### PR Implementer
This is a small checklist for the implementation details of this PR.

If there are any questions regarding code style or other conventions check out our
[summary](https://github.com/kornia/kornia/blob/master/CONTRIBUTING.rst).

- [ ] Did you discuss the functionality or any breaking changes before ?
- [ ] **Pass all tests**: did you test in local ? `make test`
- [ ] Unittests: did you add tests for your new functionality ?
- [ ] Documentations: did you build documentation ? `make build-docs`
- [ ] Implementation: is your code well commented and follow conventions ? `make lint`
- [ ] Docstrings & Typing: has your code documentation and typing ? `make mypy`
- [ ] Update notebooks & documentation if necessary

### KorniaTeam
<details>
  <summary>KorniaTeam workflow</summary>

  - [ ] Assign correct label
  - [ ] Assign PR to a reviewer
  - [ ] Does this PR close an Issue? (add `closes #IssueNumber` at the bottom if
        not already in description)

</details>

### Reviewer
<details>
  <summary>Reviewer workflow</summary>

  - [ ] Do all tests pass? (Unittests, Typing, Linting, Documentation, Environment)
  - [ ] Does the implementation follow `kornia` design conventions?
  - [ ] Is the documentation complete enough ?
  - [ ] Are the tests covering simple and corner cases ?

</details>

### CODE REVIEW

File: kornia/feature/scale_space_detector.py [hunk 1]
```diff
@@ -29,7 +29,6 @@ def _scale_index_to_scale(max_coords: torch.Tensor, sigmas: torch.Tensor, num_le
 
     # Reshape for grid shape
     B, N, _ = max_coords.shape
-    L: int = sigmas.size(1)
```
Comment (edgarriba) [inline kornia/feature/scale_space_detector.py:32]: @ducha-aiki check this and I found similar related to scale space

File: test/augmentation/test_augmentation_mix.py [hunk 2]
```diff
@@ -42,7 +42,8 @@ def test_random_mixup_p0(self, device, dtype):
             [torch.ones(1, 3, 4, device=device, dtype=dtype), torch.zeros(1, 3, 4, device=device, dtype=dtype)]
         )
         label = torch.tensor([1, 0], device=device)
-        lam = torch.tensor([0.0, 0.0], device=device, dtype=dtype)
+        # TODO(jian): where is it used ?
```
Comment (edgarriba) [inline test/augmentation/test_augmentation_mix.py:45]: @shijianjian check this
